crossfire/__init__.py

```diff
@@ -6,6 +6,12 @@
 from crossfire.clients import AsyncClient, Client  # noqa
 from crossfire.errors import NestedColumnError
 
+try:
+    from pandas import DataFrame
+except ImportError:
+    pass
+
```

Comment:
Not sure if I need to use this way... I was imagening in case a user dind't have installed Pandas...

Reply:
If you run it without having Pandas and it does not break, it is OK by now : )